### DISCUSSION TITLE
Inquiry About Timestamp Accuracy in Transcriptions

### DISCUSSION THREAD

**Discussion by kikyo1012** [deepgram/community#989] [General help]
I'm experiencing some issues with the accuracy of timestamps in the transcriptions. Specifically, I've noticed discrepancies in the start and end times while trying to segment audio based on the provided timestamps. For example, in a sample with request_id "2793acc4-2834-4434-9025-d4846ba3080f", the timestamp for the word "one" starts slightly later than the actual pronunciation begins. Consequently, if I cut the audio at the midpoint between the start of "one" and the end of the previous word, the resulting audio clip includes part of the pronunciation of "one". Could you please help me address this issue?

Thank you for your assistance.

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).

**Reply by kikyo1012** [reply]
Python
